qiita_db/util.py

```diff
@@ -926,11 +985,13 @@ def get_processed_params_tables():
     -------
     list of str
     """
-    sql = ("SELECT * FROM information_schema.tables WHERE table_schema = "
-           "'qiita' AND SUBSTR(table_name, 1, 17) = 'processed_params_'")
-
-    conn = SQLConnectionHandler()
-    return sorted([x[2] for x in conn.execute_fetchall(sql)])
+    with TRN:
+        sql = """SELECT table_name FROM information_schema.tables
+                 WHERE table_schema = 'qiita'
+                    AND SUBSTR(table_name, 1, 17) = 'processed_params_'
+                 ORDER BY table_name"""
```

Comment:
Hmmm, I like that we use ORDER BY instead of sorted, but are these two statements really the same? I guess if the tests didn't catch anything ... we should be good?